modules/internal/DefaultSparse.chpl

```diff
@@ -269,6 +293,28 @@ module DefaultSparse {
       for e in data[1..dom.nnz] do yield e;
     }
 
+    iter these(param tag: iterKind) ref where tag == iterKind.standalone {
+      const numElems = dom.nnz;
+      const numChunks = _computeNumChunks(numElems): numElems.type;
+      if debugDefaultSparse {
+        writeln("DefaultSparseArr standalone: ", numChunks, " chunks, ",
+                numElems, " elems");
+      }
+      if numChunks <= 1 {
+        for i in 1..numElems {
+          yield data[i];
+        }
+      } else {
+        coforall chunk in 1..numChunks {
+          const (startIx, endIx) =
+            _computeChunkStartEnd(numElems, numChunks, chunk);
+          for e in data[startIx..endIx] do
+            yield e;
```

Comment:
Also if they should be unified, which version is 'better'?

It seems like it'd be good to avoid having to do the slice, but I could easily be ignoring/missing someting